### PR TITLE
Add automated tests

### PR DESCRIPTION
**Pull request by dale-c-anderson**

### CODE REVIEW

File: molecule/default/converge.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+---
+- name: Converge
+  hosts: all
+  become: true
+  gather_facts: true
+  vars:
+    - package_source: 'https://github.com/wkhtmltopdf/packaging/releases/download/0.12.6-1/wkhtmltox_0.12.6-1.bionic_amd64.deb'
+    - download_dest: "{{ lookup('env','HOME') }}/wkhtmltox_0.12.6-1.bionic_amd64.deb"
+    - package_checksum: sha256:db48fa1a043309c4bfe8c8e0e38dc06c183f821599dd88d4e3cea47c5a5d4cd3
+    - xfonts_package_name: xfonts-75dpi
```
Comment (dale-c-anderson) [inline molecule/default/converge.yml]: Lines 6-10 need to go away. They'll interfere with the role's defaults.

File: defaults/Debian.yml [hunk 2]
```diff
@@ -1,7 +1,5 @@
-U---
+---
 # See https://wkhtmltopdf.org/downloads.html for checksums.
```
Comment (dale-c-anderson) [inline defaults/Debian.yml]: Please move the Debian `package_source`, `download_dest`, and `package_checksum` variables, to main.yml, in the same style that `xfonts_package_name` has been adjusted. Unfortunately there's no such thing as "include"s for defaults. It has to all happen inside main.yml.

Once they're working in main.yml, this file can be removed.

File: defaults/RedHat.yml [hunk 3]
```diff
@@ -3,5 +3,3 @@
 package_source: 'https://github.com/wkhtmltopdf/packaging/releases/download/0.12.6-1/wkhtmltox-0.12.6-1.centos8.x86_64.rpm'
```
Comment (dale-c-anderson) [inline defaults/RedHat.yml]: Ditto for RedHat variables. Once they've been re-shaped into the correct structure in main.yml, this file can be removed.

File: molecule/default/converge.yml (hunk 1 above)
Comment (Chithrak07) [reply molecule/default/converge.yml]: Done.

File: defaults/Debian.yml (hunk 2 above)
Comment (Chithrak07) [reply defaults/Debian.yml]: Done.

File: defaults/RedHat.yml (hunk 3 above)
Comment (Chithrak07) [reply defaults/RedHat.yml]: Done.

**Pull request merged**